src/compiler/config/outputs/validate-dist.ts

```diff
@@ -67,6 +67,7 @@ export const validateDist = (config: d.ValidatedConfig, userOutputs: d.OutputTar
           dir: distOutputTarget.dir,
           collectionDir: distOutputTarget.collectionDir,
           empty: distOutputTarget.empty,
+          transpileAliasedImportPaths: distOutputTarget.transpileAliasedImportPathsInCollection,
```

Comment:
is `transpile` the right verb? maybe we're transforming them, or dereferencing them? I'm not sure, maybe transpile is right

Reply:
Wasn't totally sure about this either 😅 I just carried it over from the initial issue this change was based on. "Transform" might be the better option.

Why does terminology have to be the hardest part 😂

Reply:
Before I read this thread, I was going to say "transform" might be the better option here 😆

Reply:
I think "transform" sounds good!